src/contentChunk.js

```diff
@@ -132,3 +132,12 @@ const filterPayloadType = (transaction) => {
     resolve(transaction)
   })
 }
+
+export const addOrchestrationBodies = (transaction) => {
```

Comment:
@Zooloo2014 im not sure if this function is needed. We already have a function to add the bodies to all the relevant parts for a transactions. We just need to incorporate the logic to add it still. Maybe we need to change that function name to be a bit more descriptive.
https://github.com/jembi/openhim-core-js/blob/refactoring-features/src/contentChunk.js#L114-L134

Reply:
@BMartinos Wanted to check with you if this is the correct format for the orchestrations? 
https://github.com/jembi/openhim-core-js/blob/e8d050f54de4f81802156f2caf42b665ddc94b94/test/unit/contentChunk.js#L216-L233
https://github.com/jembi/openhim-core-js/blob/e8d050f54de4f81802156f2caf42b665ddc94b94/test/unit/contentChunk.js#L253-L257

Is the orchestrations array just a collection of transactions that can be mapped to the body retrieval function?

Reply:
Ok, I see now... addOrchestrationBodies is actually the same as addBodiesToTransactions... I will fix this.

Reply:
The orchestration request/response properties has the same schema as the main transaction request/response, so your definitions look fine.